lib/utilities.js

```diff
@@ -79,13 +79,13 @@ const buildOptions = function() {
 const buildFields = (instance, field, value) => {
   // Do nothing if value is not set.
   if (value === null || value === undefined) return instance;
-  instance = instance || {};
+  instance = instance || Object.create(null);
   // Non-array fields

```

Comment:
I would add an `isSafeFromPollution(instance, field)` check right here. And move https://github.com/richardgirges/express-fileupload/blob/master/lib/processNested.js#L24-L27 into a function exported by utilities.